master/master.cfg

```diff
@@ -65,12 +69,31 @@ from custom.workers import get_workers  # noqa: E402
 settings_path = os.path.join(os.path.dirname(__file__), 'settings.yaml')
 try:
     settings = Settings.from_file(settings_path)
+    try:
+        release_sha = subprocess.check_output(["git", "rev-parse", "HEAD"], text=True)
+        sentry_sdk.init(dsn=settings.sentry_dns, release=release_sha,
+                        integrations=[SqlalchemyIntegration()])
+    except subprocess.SubprocessError:
+        log.err('WARNING: Could not get current SHA for the release!')
```

Comment:
Is this meant to wrap the `init` as well?

Reply:
Nop, thanks for the catch!